File: Iniciante/PHP/exercise_1012.php
<?php

$valores = readline("");

list($valorA, $valorB, $valorC) = sscanf($valores, "%f %f %f");

$areaTriangulo = number_format((($valorA*$valorC)/2), 3);
$areaCirculo = str_replace(',', '', (number_format((pow($valorC, 2)*3.14159), 3)));
$areaTrapezio = number_format(((($valorA+$valorB)*$valorC)/2), 3);
$areaQuadrado = str_replace(',', '', (number_format((pow($valorB, 2)), 3)));
$areaRetangulo = str_replace(',', '', (number_format(($valorA*$valorB), 3)));

print_r('TRIANGULO: '.$areaTriangulo."\n");
print_r('CIRCULO: '.$areaCirculo."\n");
print_r('TRAPEZIO: '.$areaTrapezio."\n");
print_r('QUADRADO: '.$areaQuadrado."\n");
print_r('RETANGULO: '.$areaRetangulo."\n");

File: Iniciante/Python/exercise_1012.py
valorA, valorB, valorC = [float(x) for x in input().split()]
pi = 3.14159

areaTriangulo = (valorA*valorC)/2
areaCirculo = pi*valorC**2
areaTrapezio = ((valorA+valorB)*valorC)/2
areaQuadrado = valorB**2
areaRetangulo = valorA*valorB

print(''.join(['TRIANGULO: ', "{:.3f}".format(areaTriangulo)]))
print(''.join(['CIRCULO: ', "{:.3f}".format(areaCirculo)]))
print(''.join(['TRAPEZIO: ', "{:.3f}".format(areaTrapezio)]))
print(''.join(['QUADRADO: ', "{:.3f}".format(areaQuadrado)]))
print(''.join(['RETANGULO: ', "{:.3f}".format(areaRetangulo)]))